src/Akeneo/Connectivity/Connection/front/src/infrastructure/MarketplaceSettings.tsx

```diff
@@ -0,0 +1,15 @@
+import React, {StrictMode} from 'react';
+import {HashRouter as Router} from 'react-router-dom';
+import {AkeneoThemeProvider} from './akeneo-theme-provider';
+import {withDependencies} from './dependencies-provider';
+import {Marketplace} from '../connect/pages/Marketplace';
+
+export const MarketplaceSettings = withDependencies(() => (
```

Comment:
```suggestion
export const Marketplace = withDependencies(() => (
```
This should probably just be "Marketplace" (seems like a remnant of the "ConnectionSettings" app)